test/parallel/test-runner-run.mjs

```diff
@@ -68,10 +68,18 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     const result = await run({
       files: [join(testFixtures, 'default-behavior/test/random.cjs')]
     }).compose(dot).toArray();
-    assert.deepStrictEqual(result, [
-      '.',
-      '\n',
-    ]);
+
+    if (process.stderr.isTTY) {
```

Comment:
What if instead of checking `isTTY`, you strip the problematic characters all the time.

Reply:
@cjihrig because is far as I understand we don't want to change the code implementation, we want to keep the colors and we just want to make the test ignore them

Reply:
I meant use `util.stripVTControlCharacters()` unconditionally in the test.

Reply:
oh! because I didn't know about its existence 😄 thanks!